client/gutenberg/extensions/shared/frontend-management.js

```diff
@@ -17,18 +17,19 @@ export class FrontendManagement {
 		const { selector } = options;
 		const blockClass = [ '.wp-block', name.replace( '/', '-' ) ].join( '-' );
```

Comment:
```js
[ '.wp-block', name.replace( '/', '-' ) ].join( '-' );
```

This might read more elegantly like so:

```js
`.wp-block-${ name.replace( '/', '-' ) }`
```

Or `@wordpress/blocks` has `getBlockDefaultClassName` [helper](https://github.com/WordPress/gutenberg/blob/94f54892c214ebd014ec126a9575b7e902198ad9/packages/blocks/src/api/serializer.js#L19-L32) which also applies a className filter folks might be using:

```js
getBlockDefaultClassName( name )
```

...I'm just not sure if that filter applies additional classes in some situations, which would probably be an issue for using it in `querySelectorAll()`.

Don't let this comment hold you off merging. :-)

Reply:
I'd prefer not to have to import `@wordpress/blocks` here if possible - trying to keep this pretty lean. I'll take your suggestion though, which I agree reads easier.

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/28795/commits/740dcb9a22bff5b6a7ed017298fdc444e94f39a0